packages/sanity/src/core/changeIndicators/ElementWithChangeBar.tsx

```diff
@@ -25,15 +26,16 @@ export function ElementWithChangeBar(props: {
       disabled || !isChanged ? null : (
         <ChangeBar data-testid="change-bar" $zIndex={zIndex}>
           <ChangeBarMarker data-testid="change-bar__marker" />
-
-          <ChangeBarButton
-            aria-label="Review changes"
-            data-testid="change-bar__button"
-            onClick={isReviewChangesOpen ? undefined : onOpenReviewChanges}
-            tabIndex={-1}
-            type="button"
-            $withHoverEffect={withHoverEffect}
-          />
+          <Tooltip content="Review changes" portal>
```

Comment:
This is an special button, not using the StudioUI Button, added tooltip for consistency.